CONTRIBUTING.md

```diff
@@ -13,7 +13,11 @@ section.
 ## Modules
 
 * Prefer the use of the singular form in module name (or use "multi")
-* Use the [description template](https://github.com/OCA/maintainer-tools/tree/master/template/module)
+* Use the [description template](https://github.com/OCA/maintainer-tools/tree/master/template/module) but remove sections with no meaningful content.
+* In the `__openerp__.py`  manifest file:
+  * Avoid empty keys
+  * Make sure it has the `license` key
+  * Make sure the `author` key has appended the text `,Odoo Community Association (OCA)`
```

Comment:
2015-07-02 6:49 GMT-05:00 Pedro M. Baeza <notifications@github.com>:

> s/has appended the text/has the appended text


has appended is fine.

has the appended text here do not apply.

"Ha agregado el texto " << has appended
"Tiene el el texto agregado" << pertenencia.


-- 
--------------------
Saludos Cordiales

CEO at Vauxoo <https://www.vauxoo.com> Odoo's Gold Partner.

[image: --]
Nhomar Hernandez
[image: http://]about.me/nhomar
<http://about.me/nhomar?promo=email_sig>